bin/prover-client/src/operators/checkpoint.rs

```diff
@@ -219,6 +225,24 @@ impl CheckpointOperator {
             .map_err(|e| ProvingTaskError::RpcError(e.to_string()))?
             .ok_or(ProvingTaskError::WitnessNotFound)
     }
+
+    /// Returns a reference to the internal CL (Consensus Layer) `HttpClient`.
```

Comment:
```suggestion
    /// Returns a reference to the internal CL (Consensus Layer) [`HttpClient`].
```

Awesome that you defined CL :)